src/script.c

```diff
@@ -243,8 +243,8 @@ int scriptPrepareForRun(scriptRunCtx *run_ctx,
     }
     if (client_allow_oom ||
         (!(script_flags & SCRIPT_FLAG_EVAL_COMPAT_MODE) && (script_flags & SCRIPT_FLAG_ALLOW_OOM))) {
-        /* Note: we don't need to test the no-writes flag here and set this run_ctx flag,
-         * since only write commands can are deny-oom. */
+        /* Note: we don't need to test the no-writes flag here when we set this run_ctx flag,
+         * since only write commands are deny-oom. */
```

Comment:
the `can are` part tripped a rule. It took me a while to be able to get some understanding of what this was trying to say. I'm not confident about my corrections here.